include/dpp/guild.h

```diff
@@ -520,6 +520,14 @@ class DPP_EXPORT guild_member : public json_interface<guild_member> {
 	 */
 	bool has_rejoined() const;
 
+	/**
+	 * @brief Returns true if the user is the guild owner.
+	 * @return true if the user is the guild owner.
+	 * @return false if the user is not the guild owner or the guild is not in the cache.
+	 * @note If dpp cache is disabled (which should be enabled by default) this function will always return false.
```

Comment:
this should be reworded:
```cpp
/*
 * @note If the guild cache is disabled in the cluster constructor, this function will always return false.
 */
```
we dont need to tell them that its on by default, other docs indicate this